File: WorldFeed/Server/src/WorldFeed.Streams/Model/AccountActivity/BaseAccountActivityMessageEventDTO.cs
namespace WorldFeed.Streams.Model.AccountActivity
{
    using System.Collections.Generic;
    using Newtonsoft.Json;

    using WorldFeed.Common.DTO;
    using WorldFeed.Common.Models.Properties;

    public class BaseAccountActivityMessageEventDTO
    {
        [JsonProperty("apps")]
        public Dictionary<string, App> Apps { get; set; }

        [JsonProperty("users")]
        public Dictionary<string, UserDTO> UsersById { get; set; }
    }
}

File: Sharebook/Client/src/app/streams/Model/AccountActivity/BaseAccountActivityMessageEventDTO.ts
import Dictionary from "../../../c#-objects/TypeScript.NET-Core/packages/Core/source/Collections/Dictionaries/Dictionary";
import {App} from "../../../core/Core/Models/Properties/App";
import {UserDTO} from "../../../core/Core/DTO/UserDTO";

export class BaseAccountActivityMessageEventDTO {
  // [JsonProperty("apps")]
  public Apps: Dictionary<string, App>;

  // [JsonProperty("users")]
  public UsersById: Dictionary<string, UserDTO>;
}
